gclient/src/api/calls.rs

```diff
@@ -134,7 +178,36 @@ impl GearApi {
         }
     }
 
-    /// `pallet_gear::create_program` with `impl Encode` type in payload.
+    /// Create a new program from a previously uploaded code identified by
+    /// [`CodeId`](https://docs.gear.rs/gear_core/ids/struct.CodeId.html) and
+    /// initialize it with an encoded `payload`.
+    ///
+    /// Sends the
+    /// [`pallet_gear::create_program`](https://docs.gear.rs/pallet_gear/pallet/struct.Pallet.html#method.create_program)
+    /// extrinsic.
+    ///
+    /// Parameters:
+    ///
+    /// - `code_id` is the code identifier that can be obtained by calling the
+    ///   [`upload_code`](Self::upload_code) function;
+    /// - `salt` is the arbitrary data needed to generate an address for a new
+    ///   program (control of salt uniqueness is entirely on the function
+    ///   caller’s side);
+    /// - `payload` contains encoded data to be processed in the `init` function
+    ///   of the newly deployed "child" program;
+    /// - `gas_limit` is the maximum gas amount allowed to spend for the program
+    ///   creation and initialization;
+    /// - `value` to be transferred to the program's account during
+    ///   initialization.
+    ///
+    /// # See also
+    ///
+    /// - [`create_program_bytes`](Self::create_program_bytes) function
+    ///   initializes a newly created program with a byte slice payload.
+    /// - [`upload_code`](Self::upload_code) function uploads a code and returns
+    ///   its identifier.
+    /// - [`upload_program`](Self::upload_program) function uploads a new
+    ///   program and initialize it.
     pub async fn create_program(
```

Comment:
I'd ref FN with FN_bytes by doc-ref as it did for gcore etc. To avoid useless copy-paste

Reply:
Done